AgentDiscoveries-Frontend/app/src/components/submit-forms/submit-location-report.jsx

```diff
@@ -59,9 +61,19 @@ export default class LocationReportSubmit extends React.Component {
                             onChange={this.onStatusChange}
                             id="status-input"/>
                     </FormGroup>
+                    <FormGroup>
+                        <ControlLabel>Title</ControlLabel>
+                        <FormControl type='text' required
+                            componentClass='textarea'
+                            placeholder='Write title'
+                            value={this.state.reportTitle}
+                            onChange={this.onReportTitleChange}
+                            id="title-input"/>
+                    </FormGroup>
                     <FormGroup>
                         <ControlLabel>Report</ControlLabel>
                         <FormControl type='text' required
+                            maxlength='30'
```

Comment:
It looks like this is on the wrong field - isn't this limiting the report body length rather than the title?